docs/docs/hello-world.md

```diff
@@ -3,6 +3,25 @@ id: hello-world
 title: Hello World
 permalink: docs/hello-world.html
 prev: installation.html
+next: introducing-jsx.html
 ---
 
-TODO
+The smallest React example looks like this:
+
+```js
+let element = <h1>Hello, world!</h1>;
+let domNode = document.getElementById('root');
+ReactDOM.render(element, domNode);
+```
+
+**[Try it on Codepen.](http://codepen.io/gaearon/pen/ZpvBNJ?editors=0010)**
+
+It renders a header saying "Hello World" on the page.
+
+There are a few things to unpack here:
```

Comment:
People are likely coming to the documentation because they want to write things using React, not because they want to understand how the internals of React are working.

What do you think about making the first few docs page as a progressive disclosure of all the concepts through examples of things you can do with React instead of being three big sections on each concept.